### DISCUSSION TITLE
Add traceback_with_variables as an optional part of issue template

### DISCUSSION THREAD

**Discussion by derlih** [aio-libs/aiohttp#5300] [Ideas] [answered]
Hi

Today I found an interesting python module: [traceback_with_variables](https://github.com/andy-landy/traceback_with_variables).
Probably it can help us to debug issues easier:
```
pip install traceback-with-variables
traceback-with-variables ...script/tool/module with arguments...
```
Then the stacktrace will contain the function call arguments
![](https://raw.githubusercontent.com/andy-landy/traceback_with_variables/master/color_schemes.png)

What do you think about it?

**Comment by asvetlov** [comment, marked as answer]
Looks pretty good, please go ahead

**Comment by andy-landy** [comment]
Hi, the module author and maintainer here. Today I saw an unexpected surge in stars and traffic ;) Anyway, I wanted to note that I have some time resources for prompt feature adding and tailoring if anyone requests something. There's a list of new features to be added in the nearest time, some of them are mentioned in issues, some of them are implemented in `next_version` branch. 

If anyone finds this module practical in their project, I'm fine to help, collaborate, add and tailor.
